Content.Server/Administration/Logs/Converters/CargoOrderDataStringRepresentationConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text.Json;
+using Content.Shared.Cargo.Orders;
+
+namespace Content.Server.Administration.Logs.Converters;
+
+[AdminLogConverter]
+public sealed class CargoOrderDataStringRepresentationConverter : AdminLogConverter<CargoOrderStringRepresentation>
+{
+    public override void Write(Utf8JsonWriter writer, CargoOrderStringRepresentation value, JsonSerializerOptions options)
+    {
+        writer.WriteStartObject();
+
+        writer.WriteNumber("orderId", value.OrderId);
+        writer.WriteString("productName", value.ProductName);
+        if (value.EntityUid != null)
+            writer.WriteNumber("entityUid", value.EntityUid.Value.GetHashCode()); // How do I make this use EntityUidConverter.cs instead?
```

Comment:
make a WriteEntityUid method? (if thats the problem, idk if thats your actual issue here)